### ISSUE TITLE
Add Action Network events endpoints

### ISSUE THREAD

**Issue by esnowrackley** [move-coop/parsons#520]
Currently Parsons doesn't include the events endpoints for Action Network. ~This prevents us from specifying page size and page number and makes pulling down events complex and difficult to parse.~

By adding these, we could use a function similar to `get_people` to pull down events specifying the page number and page size as args. Right now we can accomplish that by invoking `_get_page`, but not as its own function.

**Comment by schlich** [comment]
happy to submit a PR for this.  Can you elaborate on what you mean by this? 

> This prevents us from specifying page size and page number

**Comment by esnowrackley** [comment]
Whoops-- that isn't super descriptive. :) See revised above.

**Comment by stevans** [comment]
Our organization would love Action Network events endpoints!

**Comment by schlich** [comment]
Still working on a PR for this, but I want to clean up the testing framework to be a little bit more versatile here (e.g. integrating pytest and factoryboy) so we can ideally knock out the rest of the API. It's already getting pretty busy.  Are there any preferred architectures being used for this elsewhere in Parsons?

**Comment by shaunagm** [comment]
Hi @schlich! Are you still planning to submit a PR? I know it's been ages and ages, so no worries if you're not.

**Comment by schlich** [comment]
> Hi @schlich! Are you still planning to submit a PR? I know it's been ages and ages, so no worries if you're not.

Yes, this is absolutely still on my radar! Thank you for the nudge.  I've done a lot of work with the Action Network API since my last comment (and am a much more experienced tester) so I should be able to make quick work of it.

**Comment by shaunagm** [comment]
Excellent news. :)

Fyi we've got a contributor call tomorrow at 2pm ET, feel free to join us! lmk if you need join info.

**Comment by shaunagm** [comment]
Hi @schlich, what's the status on this? Happy to pair on it if that's helpful.

**Comment by schlich** [comment]
It's still on my radar, and I apologize for not following up/following through sooner.  I got partway through a big refactoring before realizing I needed to break it up into smaller chunks.  I'm also keeping eyes on a colleague developing a version of the api client published under the `osdi` package (pretty primitive still, just wanna make sure we don't duplicate work).

I would be very happy to pair on this if only to keep me accountable :) I still think we can make fairly quick work of it.
